src/CacheTool/Adapter/FastCGI.php

```diff
@@ -21,6 +21,11 @@ class FastCGI extends AbstractAdapter
      */
     protected $client;
 
+    protected $possibleSocketFiles = [
+        '/var/run/php5-fpm.sock',
+        '/var/run/php/php7.0-fpm.sock'
```

Comment:
sure, I pushed a bit too fast.
I'll change for both `['/var/run/php*.sock', '/var/run/php/php7.0-fpm.sock']`